src/server/pfs/drive/driver.go

```diff
@@ -947,7 +947,7 @@ func (d *driver) inspectFile(file *pfs.File, filterShard *pfs.Shard, shard uint6
 	for commit != nil && (from == nil || commit.ID != from.ID) {
 		diffInfo, ok := d.diffs.get(client.NewDiff(commit.Repo.Name, commit.ID, shard))
 		if !ok {
-			return nil, nil, fmt.Errorf("diff %s/%s/%d not found", commit.Repo.Name, commit.ID, shard)
+			return nil, nil, pfsserver.ErrCommitNotFound
```

Comment:
Ok talking w this more, we will report all the info in all the cases.

I'll use this opportunity to make the error reporting more consistent across the board. We still have some error strings in lieu of using the preset errors. And we don't have errors for missing job / etc.